Sources/Nimble/Matchers/PostNotification.swift

```diff
@@ -27,6 +27,7 @@ internal class NotificationCollector {
 
 private let mainThread = pthread_self()
 
+@available(*, deprecated)
 public func postNotifications<T>(
```

Comment:
If it is marked deprecated, there is no non-deprecated alternative (overload).

Reply:
So I guess we need to either have an overload that doesn't use `Matcher`, or have this one not use `Matcher` ?